packages/remix/test/unit.get-path-from-route.test.ts

```diff
@@ -104,6 +104,14 @@ describe('getPathFromRoute()', () => {
       parentId: 'root',
       file: 'routes/($lang)/$pid.tsx',
     },
+    'routes/admin.(lol)': {
```

Comment:
**question (non-block):** I know Remix supports Dot Delimeters, but is the dot required or is a forward slash also acceptable?

Reply:
Forward slash also works. The dot syntax is to allow a route to skip using the parent layout route (so would use `root.tsx` as the layout instead of `routes/admin.tsx` in this case).